modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -2,162 +2,129 @@
 
 = Setting up {org-name} {product-name} on OpenShift
 
-To use {product-name}, you must run OpenShift and install an add-on which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
-
-NOTE: {org-name} recommends running OpenShift using the method described in this document. However, there are many ways to run OpenShift, and {product-name} is compatible with most of these methods. There is a known issue with Minishift that makes it unsuitable for mobile development. This issue is described in link:https://github.com/minishift/minishift/issues/1287[Issue 1287, window="_blank"].
+To use {product-name}, you must have OpenShift instance and run installation script which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
 
 [[prerequisites]]
 == Prerequisites
 
-* MacOS or Linux
-+
-NOTE: Due to an link:https://github.com/openshift/origin/issues/18747[OpenShift bug], {product-name} will not run on RHEL/CentOS 7.4 or RHEL/CentOS 7.5.
-
-* A system running OpenShift `oc cluster up` as described in link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#prerequisites[Local Cluster Management, window="_blank"]
-+
-NOTE: Installation on macOS with Docker Machine has not been tested, use the link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#macos-with-docker-for-mac[Docker for Mac] instructions when installing OpenShift.
+* OpenShift 3.11 instance
 +
-** Use link:https://www.openshift.org/download.html[OpenShift client tools, window="_blank"] version 3.9
+** This instace must have service catalog, ansible service broker and template service broker services installed.
```

Comment:
@jhellar do we cover any of the steps using the scripts in the mdc. Could we point the user there in order to get the local setup done?

Reply:
sorry I see that that step is covered further down.

Reply:
Capital Ansible

Reply:
instaNce

Reply:
@jhellar small typo `instace` -> `instance`